src/utils/path.js

```diff
@@ -10,29 +10,24 @@ import _ from 'lodash'
  */
 function findRelativePath (path, index = 0) {
```

Comment:
Since you're in here, can you add an example of possible paths this might encounter and what the output is. Every time I read this, I get confizzled.